scripts/run-with-strimzi.sh

```diff
@@ -37,17 +55,54 @@ if [ $MINIKUBE_MEM_EXIT -eq 0 ]; then
 else
   echo "no minikube memory configuration, defaulting to 4096M"
 fi
-INSTALL_DIR="sample-install"
 ${MINIKUBE} start "${MINIKUBE_CONF}"
-${KUBECTL} create namespace kafka || true
+
+NAMESPACE=kafka
+
+# Prepare kustomize overlay
+cp -r "${SAMPLE_DIR}" "${KUSTOMIZE_TMP}"
+OVERLAY_DIR=$(find "${KUSTOMIZE_TMP}" -type d -name minikube)
+
+if [[ ! -d "${OVERLAY_DIR}" ]]; then
+     echo "$0: Cannot find minikube overlay within sample."
+     exit 1
+fi
+
+pushd "${OVERLAY_DIR}"
+${KUSTOMIZE} edit set namespace ${NAMESPACE}
+if [[ "${QUAY_ORG}" != "${DEFAULT_QUAY_ORG}" ]]; then
+  ${KUSTOMIZE} edit set image "quay.io/kroxylicious/${DEFAULT_QUAY_ORG}=quay.io/${QUAY_ORG}/kroxylicious"
```

Comment:
I think these two lines aren't quite right, `DEFAULT_QUAY_ORG` should be `kroxylicious` and the set could be: `edit set image "quay.io/${DEFAULT_QUAY_ORG}/kroxylicious-developer=quay.io/${QUAY_ORG}/kroxylicious`.

Reply:
Thanks, good catch.  I was going cross-eyed